runtime/nls/cfre/cfrerr.nls

```diff
@@ -1668,3 +1668,56 @@ J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.explanation=Please consult the Java Virtual Ma
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.system_action=The JVM will throw a verification or class loading-related exception such as java.lang.ClassFormatError.
 J9NLS_CFR_ERR_INIT_ON_VALUE_CLASS.user_response=Contact the provider of the class file for a corrected version.
 # END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_MAJOR_VERSION2=bad major version; class=%2$.*1$s, offset=%3$u, major.minor=%4$u.%5$u, maximum major value=%6$u.0
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_3=6
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_4=62
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_5=0
+J9NLS_CFR_ERR_MAJOR_VERSION2.sample_input_6=61
+J9NLS_CFR_ERR_MAJOR_VERSION2.explanation=The major version in the classfile is unsupported on the VM rejecting the classfile.
+J9NLS_CFR_ERR_MAJOR_VERSION2.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MAJOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+
+# END NON-TRANSLATABLE
+
+
+J9NLS_CFR_ERR_MINOR_VERSION2=bad minor version of class %2$.*1$s. Got minor version .%3$u, but .0 is the only valid minor version for the major version %4$u; offset=%5$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_1=4
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_2=main
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_3=1
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_4=63
+J9NLS_CFR_ERR_MINOR_VERSION2.sample_input_5=6
+J9NLS_CFR_ERR_MINOR_VERSION2.explanation=An invalid minor version has been specified.
+J9NLS_CFR_ERR_MINOR_VERSION2.system_action=The JVM will throw a verification or classloading related exception such as java.lang.ClassFormatError.
+J9NLS_CFR_ERR_MINOR_VERSION2.user_response=Contact the provider of the classfile for a corrected version.
+
+# END NON-TRANSLATABLE
+
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED=version %1$u.%2$u of class %4$.*3$s is a preview version but preview is not enabled, please enable it with --enable-preview; offset=%5$u
+# START NON-TRANSLATABLE
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.sample_input_1=62
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.sample_input_2=65535
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.sample_input_3=4
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.sample_input_4=main
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.sample_input_5=6
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.explanation=Classfiles with minor version 65535 must have the same major version as the VM runtime version and preview must be enabled
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.system_action=The JVM will throw a verification or classloading related exception.
+J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.user_response=Run with the correct Java runtime version and use --enable-preview, or re-compile the code with the same version of the runtime.
```

Comment:
The addition of "or re-compile the code with the same version of the runtime" isn't applicable to this error, where you need to add "--enable-preview".

Reply:
So I should change it to something like "J9NLS_CFR_ERR_PREVIEW_VERSION_NOT_ENABLED.user_response=Enable preview version with --enable-preview."?

Reply:
You can put it back to how it was, there was nothing wrong with this one.